Ship_Game/GameScreens/ShipDesign/ShipDesignScreen.cs

```diff
@@ -640,5 +646,21 @@ void RefreshHullSelectList()
                 }
             }
         }
+
+        public void ResetMusic()
+        {
+            GameAudio.ConfigureAudioSettings(GlobalStats.MusicVolume, GlobalStats.EffectsVolume);
+            GameAudio.StopGenericMusic(fadeout: false);
+            ScreenManager.Music.Stop();
+      
+            if (GlobalStats.Defaults.CustomMenuMusic.NotEmpty())
+            {
+                ScreenManager.Music = GameAudio.PlayMusicFile(GlobalStats.Defaults.CustomMenuMusic);
+            }
+            if (ScreenManager.Music.IsStopped)
+            {
+                ScreenManager.Music = GameAudio.PlayMusic("ShipyardTheme");
+            }

```

Comment:
```suggestion
            if (GlobalStats.Defaults.CustomMenuMusic.NotEmpty())
                ScreenManager.Music = GameAudio.PlayMusicFile(GlobalStats.Defaults.CustomMenuMusic);
                
            if (ScreenManager.Music.IsStopped)
                ScreenManager.Music = GameAudio.PlayMusic("ShipyardTheme");
```
If you If-Else with 1 line block, its better not to use curly brackets